### PR TITLE
Fix bug found by Lenard about element names with extra "s"

### PR DESCRIPTION
**Pull request by mhdirkse**

### CODE REVIEW

File: core/src/main/java/nl/nn/adapterframework/doc/model/ConfigChild.java [hunk 1]
```diff
@@ -32,6 +37,12 @@
 			.thenComparing(c -> c.getElementRole().getRoleName())
 			.thenComparing(c -> c.getElementRole().getElementType().getFullName());
 
+	private static final Comparator<ConfigChild> INVERSE_ALLOW_MULTIPLE =
```
Comment (gvanbrakel) [inline core/src/main/java/nl/nn/adapterframework/doc/model/ConfigChild.java:40]: Misschien deze 'SINGLE_ELEMENT_ONLY' noemen?

File: core/src/main/java/nl/nn/adapterframework/doc/model/ElementChild.java [hunk 2]
```diff
@@ -102,13 +114,18 @@ void calculateOverriddenFrom() {
...
+					technicalOverride = true;
+				}
 				log.trace("{} [{}] of FrankElement [{}] has overriddenFrom = [{}]",
 						() -> getClass().getSimpleName(), () -> toString(), () -> owningElement.getFullName(), () -> overriddenFrom.getFullName());
 				return;
 			}
 		}
 	}
 
+	abstract boolean checkOverrideMeaningful(ElementChild overriddenFrom);
```
Comment (gvanbrakel) [inline core/src/main/java/nl/nn/adapterframework/doc/model/ElementChild.java:127]: Misschien deze 'overrideIsMeaningful()' noemen?

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

**Comment by mhdirkse** [comment]
I need to add more logging. Now, the log does not show anything about filtering config children as implemented here.

File: core/src/main/java/nl/nn/adapterframework/doc/model/ElementChild.java (hunk 2 above)
Comment (mhdirkse) [reply core/src/main/java/nl/nn/adapterframework/doc/model/ElementChild.java]: Gedaan.

File: core/src/main/java/nl/nn/adapterframework/doc/model/ConfigChild.java (hunk 1 above)
Comment (mhdirkse) [reply core/src/main/java/nl/nn/adapterframework/doc/model/ConfigChild.java]: Gedaan.

**Comment by mhdirkse** [comment]
Gerrit, je hebt mij gevraagd om de extra functionaliteit van deze pull request te unit testen. Dat zal ik doen.

**Comment by mhdirkse** [comment]
I can make the unit tests stronger. One moment please before merging, I fix that now.

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**